primefaces/src/main/resources/META-INF/resources/primefaces/captcha/captcha.js

```diff
@@ -31,6 +31,7 @@ PrimeFaces.widget.Captcha = PrimeFaces.widget.BaseWidget.extend({
     init: function(cfg) {
         this._super(cfg);
         this.cfg.language = this.cfg.language||'en';
+        this.cfg.theme = this.cfg.theme || PrimeFaces.env.getThemeContrast();
```

Comment:
The only thing I was doubting if we shouldn't do something like
```js
if (this.cfg.theme === 'auto') {
    this.cfg.theme = PrimeFaces.env.getThemeContrast();
}
```

Reply:
Great idea!